src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -150,14 +173,9 @@ public long getTimeUntilNextRun()
      * {@inheritDoc}
      */
     @Override
-    public void incomingPacket(
-            long arrivalTimeMs,
-            int payloadSize,
-            int ssrc,
-            long rtpTimestamp,
-            boolean wasPaced)
+    public RawPacket reverseTransform(RawPacket pkt)
     {
-        Integer ssrc_ = ssrc;
+        Integer ssrc_ = pkt.getSSRC();
```

Comment:
Just gave it a try and, if we want to do this properly, the changes are not contained in the method, it changes the remote bitrate estimator interface. I would leave it for another PR.